paddle/fluid/memory/malloc_test.cu

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+//
```

Comment:
Add a unittest when multi-threads try to call `memory::Alloc(dev_ctx, size)` and launch CUDA kernels?

Reply:
Done